src/LinuxTracing/TracerImpl.cpp

```diff
@@ -299,7 +299,7 @@ void TracerImpl::AddUprobesFileDescriptors(
     } else {
       uprobes_ids_.insert(stream_id);
     }
-    tracing_fds_.push_back(fd);
+    tracing_fds_by_type_["uprobe"].push_back(fd);
```

Comment:
Can you add constants for the strings? like constexpr kUprobeId = "uprobe". Same below.